gen/pb-typescript/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
Why is this needed?

Reply:
This ensures that the `LICENSE` gets included in the npm package.